crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -17,10 +17,16 @@ use bevy_render::{
 #[bind_group_data(StandardMaterialKey)]
 #[uniform(0, StandardMaterialUniform)]
 pub struct StandardMaterial {
```

Comment:
I would like to see some struct-level docs about "please don't try to set the texture values yourself, they're automatically created in SYSTEM"

Reply:
I'm not sure I follow. Do you mean I should point out to the GLTF (or FBX) loader(s)?

Reply:
Oh! I think I see my confusion here. Can you set the value of `base_color` and `base_color_texture` independently? I was thinking the latter was derived, but I don't think that's actually true.